packages/web-app-external/src/App.vue

```diff
@@ -105,7 +104,7 @@ export default defineComponent({
 
         const query = stringify({
           file_id: fileId,
-          lang: language.current,
+          lang: currentLanguage,
```

Comment:
This was probably the reason why there was a language object and the return value of useGettext() wasn't destructured ... this loses reactivity

Reply:
yep, sure. Could you check the PR again please?